modules/updating-lmeval-job-configuration-using-the-console.adoc

```diff
@@ -0,0 +1,82 @@
+:_module-type: PROCEDURE
+
+ifdef::context[:parent-context: {context}]
+[id="updating-lmeval-job-configuration-using-the-console_{context}"]
+= Updating LMEval job configuration using the console
```

Comment:
I think we usually say "web console" vs console but unsure if this is universal. Was in OCP I believe :)

Reply:
nothing in the IBM style guide I could find so fee free to ignore